generation/day_ahead.py

```diff
@@ -155,7 +156,7 @@ def get_series_for_sensor(sensor):
             series.name = "event_value"  # required by timely_beliefs, TODO: check if that still is the case, see https://github.com/SeitaBV/timely-beliefs/issues/64
             bdf = BeliefsDataFrame(
                 series,
-                source=data_source,
+                source=entsoe_data_source if not sensor.name == "CO2 intensity" else derived_data_source,
```

Comment:
We have a data structure in generation/ utils.py for sensors we handle.
Maybe a flag for derived or not could be placed there, and then be used here?
This might work better when this plugin grows.